x/interchainstaking/keeper/withdrawal_record.go

```diff
@@ -276,7 +276,10 @@ func (k *Keeper) UpdateWithdrawalRecordsForSlash(ctx sdk.Context, zone *types.Zo
 				return true
 			}
 
-			newAmount := distAmount.Quo(delta).TruncateInt()
+			newAmount := sdkmath.ZeroInt()
+			if !delta.IsZero() {
+				newAmount = distAmount.Quo(delta).TruncateInt()
+			}
```

Comment:
@coderabbitai we always use truncate to be conversative, to ensure that any values are rounded down, so that the amount claimed by the user is always <= the amount in the account. If we rounded up, there would be insufficient funds in the account when it came to distribute funds.